src/components/pages/users.js

```diff
@@ -69,6 +90,31 @@ class Users extends React.Component {
         render: this.renderRoleTags
       }
     ];
+
+    this.externalUserTableColumns = [
+      {
+        title: () => this.props.t('username'),
+        dataIndex: 'username',
+        key: 'username',
+        sorter: firstBy('username'),
+        render: this.renderUsername
+      }, {
+        title: () => this.props.t('email'),
+        dataIndex: 'email',
+        key: 'email',
+        sorter: firstBy('email')
+      }, {
+        title: () => this.props.t('id'),
```

Comment:
Why do we need this column? I see no value for the user.

Reply:
True, removed